remindme-common/src/main/java/it/andreafailli/remindme/common/models/Reminder.java

```diff
@@ -1,12 +1,12 @@
 package it.andreafailli.remindme.common.models;
 
-import java.util.Date;
+import java.time.LocalDateTime;
 
 public class Reminder extends BaseEntity {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)